insights/parsers/lssap.py

```diff
@@ -59,8 +59,14 @@ def __init__(self, *args, **kwargs):
         super(Lssap, self).__init__(*args, **kwargs)
 
     def parse_content(self, content):
-        # remove lssap version and bar text from content
-        clean_content = content[2:-1]
+        # Ensure we only parse content after the lssap version line. Default the
+        # starting index to 2 for expected lssap output
+        start = 2
+        for row in content:
+            if 'lssap' and 'version' in row:
+                start = content.index(row) + 2
+        clean_content = content[start:-1]
+        print(clean_content)
         if len(clean_content) > 0 and "SID" in clean_content[0]:
             self.data = parse_delimited_table(clean_content, delim='|', header_delim=None)
```

Comment:
Firstly, you'll need to remove your debugging `print` statement :-)

Secondly, just wondering if you can use the `heading_ignore` option to `parse_delimited_table` to do this kind of search automatically.  `heading_ignore` actually indicates the line of the header (rather than ignoring it), so you could do something like:
```python
self.data = parse_delimited_table(content, delim='|', header_delim=None, heading_ignore=['  SID'])
```
That might be an easier way of getting the header line right 'automatically'.